ppx/query.mll

```diff
@@ -102,7 +109,24 @@ let parse query =
   let buf = Buffer.create (String.length query) in
   main_parser buf [] [] lexbuf
 
-let explain_parse_error = function
+let remove_duplicates params =
+  let rec loop dict accum = function
+    | [] ->
+        Ok (List.rev accum)
+    | hd :: tl ->
```

Comment:
Destructuring `hd` here would avoid writing `hd.name` a dozen times down below.

Reply:
Fair point. I'll make the change.